utils/rtty/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2018 Jianhui Zhao
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtty
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/zhaojh329/rtty.git
+PKG_SOURCE_VERSION:=84d89b248f4c07bfd8140f3642ba6d0fae979f8e
+PKG_MIRROR_HASH:=ec38a1d9b32399faa596c1023ab6d8141ac64f574c6630abe2be558b96b7e095
+
+PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_SOURCE_SUBDIR)
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_MAINTAINER:=Jianhui Zhao <jianhuizhao329@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/rtty/default
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Terminal
+  TITLE:=A reverse proxy WebTTY
+  DEPENDS:=+libblobmsg-json
+endef
+
+define Package/rtty/default/description
+A reverse proxy WebTTY. It is composed of the client and the server.
+It is composed of the client and the server. You can access any of
+your terminals through a web browser based on the device ID you set.
+endef
+
+define Package/rtty-nossl
+  $(Package/rtty/default)
+  TITLE += (NO SSL)
+  DEPENDS += +libuwsc-nossl
+  VARIANT:=nossl
+endef
+
+define Package/rtty-openssl
+  $(Package/rtty/default)
+  TITLE += (openssl)
+  DEPENDS += +libuwsc-openssl
+  VARIANT:=openssl
+endef
+
+define Package/rtty-wolfssl
+  $(Package/rtty/default)
+  TITLE += (wolfssl)
+  DEPENDS += +libuwsc-wolfssl
+  VARIANT:=wolfssl
+endef
+
+define Package/rtty-mbedtls
+  $(Package/rtty/default)
+  TITLE += (mbedtls)
+  DEPENDS += +libuwsc-mbedtls
+  VARIANT:=mbedtls
+endef
+
+Package/rtty-nossl/description = $(Package/rtty/default/description)
+Package/rtty-openssl/description = $(Package/rtty/default/description)
+Package/rtty-wolfssl/description = $(Package/rtty/default/description)
+Package/rtty-mbedtls/description = $(Package/rtty/default/description)
+
+define Package/rtty/default/install
+	$(INSTALL_DIR) $(1)/usr/sbin $(1)/etc/init.d $(1)/etc/config
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/rtty $(1)/usr/sbin
+	$(INSTALL_BIN) ./files/rtty.init $(1)/etc/init.d/rtty
+	$(INSTALL_CONF) ./files/rtty.config $(1)/etc/config/rtty
+endef
+
+Package/rtty-nossl/install = $(Package/rtty/default/install)
+Package/rtty-openssl/install = $(Package/rtty/default/install)
+Package/rtty-wolfssl/install = $(Package/rtty/default/install)
+Package/rtty-mbedtls/install = $(Package/rtty/default/install)
+
+$(eval $(call BuildPackage,rtty-nossl))
+$(eval $(call BuildPackage,rtty-mbedtls))
+$(eval $(call BuildPackage,rtty-wolfssl))
+$(eval $(call BuildPackage,rtty-openssl))
+
```

Comment:
Please also add `CONFLICTS`, examples can be taken from `package/network/services/hostapd/Makefile`

Reply:
Fixed.